refinery/ui/source/js/commons/services/tool-definitions.js

```diff
@@ -14,7 +14,7 @@
         query: {
           method: 'GET',
           isArray: true,
-          params: { dataSetUuid: '' }
+          params: { data_set_uuid: '' }
```

Comment:
Sure, since it's a url query. Worth noting for json it's camelCase styling. https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format

Reply:
Yeah I wasn't sold on one way vs. the other, but I can change the backend code to work with `dataSetUuid`